src/api/middleware/auth/authentication_handler.go

```diff
@@ -3,28 +3,25 @@ package auth
 import (
 	"fmt"
 	"github.com/arpb2/C-3PO/src/api/auth"
-	"github.com/arpb2/C-3PO/src/api/auth/jwt"
 	"github.com/arpb2/C-3PO/src/api/controller"
 	"github.com/gin-gonic/gin"
 	"net/http"
 	"strconv"
 )
 
-var TokenHandler auth.TokenHandler = jwt.TokenHandler{
-	Secret: jwt.FetchJwtSecret(),
+type AuthenticationStrategy interface {
```

Comment:
👍

Reply:
Te va a ser mucho mas cómodo testear asi con la lib que pasé en el otro PR

Reply:
Fixed in b11aa958aed8f3a5ac817b225dc757761272b54d